### ISSUE TITLE
Both tabSetEnableClose and enableClose in tabset do not work

### ISSUE THREAD

**Issue by JingLiNaomi** [caplin/FlexLayout#30]
Hi,

I am trying to add close icon to tabset, when global tabSetEnableTabStrip is false, but both tabSetEnableClose and enableClose in tabset do not work. Please see my code below:
https://jsfiddle.net/rmf3hzmf/95/

Am I using it in wrong way? Thanks

**Comment by nealus** [comment]
Yes you are right, that is not implemented, I've removed it from the code/readme
